winlocalprocessspawner/winlocalprocessspawner.py

```diff
@@ -44,7 +44,13 @@ def get_env(self):
         token = None
 
         cmd.append(sys.executable)
-        py_scripts_dir = os.path.join(os.path.dirname(sys.executable), 'Scripts')
+        #in virtualenv, exe is in Scripts folder already vs main environment where exe is a python root
+        if hasattr(sys, 'real_prefix'):

```

Comment:
The Spawner already has a variable (c.Spawner.cmd) to define custom paths to jupyterhub-singleuser, which is used for virutalenv support. The code is not currently using the variable....but I think it should, following suit with what the LocalProcessSpawner does.

Reply:
Ah yeah, makes more sense than hardcoding the difference. I'll look into it.